tests/integration_tests/tests/agentless_tests/test_plugins.py

```diff
@@ -30,6 +34,12 @@ def test_get_plugin_by_id(self):
             put_plugin_response = test_utils.upload_mock_plugin(
                     TEST_PACKAGE_NAME,
                     TEST_PACKAGE_VERSION)
+            ex = self._get_latest_execution('install_plugin')
```

Comment:
I guess this section is not relevant anymore

Reply:
Why not?